specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/policyStates.json

```diff
@@ -1161,22 +1161,19 @@
           }
         }
       }
-    },
-    "/{nextLink}": {
+    }
+  },
+  "x-ms-paths": {
+    "/{nextLink}?Next link op for policy states": {
```

Comment:
Basically, we're hacking swagger here.

In your current PR, you have two operations defined with path `"/{nextLink}`. Since these 2 files are combined under the same client, swagger has an issue with this. By switching to `x-ms-paths`, autorest basically ignores everything after the question mark, so I'm able to hack them having different endpoints in swagger. I also changed the operation name since they both had the same one